apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx

```diff
@@ -0,0 +1,326 @@
+
+        setIdPListRequestLoading(true);
+
+        getIdentityProviderList(null, null, null)
+            .then((response: IdentityProviderListResponseInterface) => {
+                setIdPList(response);
+            })
+            .catch((error: Promise<void>) => {
+                handleGetIDPListCallError(error);
+            })
+            .finally(() => {
+                setIdPListRequestLoading(false);
+            });
+    };
+
+    /**
+     * Check whether IDP name is already exist or not.
+     *
+     * @param value - IDP name.
+     * @returns error msg if name is already taken.
+     */
+    const idpNameValidation = (value: string): string => {
+
+        let nameExist: boolean = false;
+
+        if (idpList?.count > 0) {
+            idpList?.identityProviders.map((idp: IdentityProviderTemplateInterface) => {
+                if (idp?.name === value) {
+                    nameExist = true;
+
+                }
+            });
+        }
+        if (nameExist) {
+            return t("console:develop.features." +
+                "authenticationProvider.forms.generalDetails.name." +
+                "validations.duplicate");
+        }
+    };
+
+    /**
+     * Validates the Form.
+     *
+     * @param values - Form Values.
+     * @returns Form validation.
+     */
+    const validateForm = (values: HyprAuthenticationProviderCreateWizardFormValuesInterface):
+        HyprAuthenticationProviderCreateWizardFormValuesInterface => {
+
+        const errors: HyprAuthenticationProviderCreateWizardFormValuesInterface = {
+            apiToken: undefined,
+            appId: undefined,
+            baseUrl: undefined,
+            name: undefined
+        };
+
+        if (!values.name) {
+            errors.name = t("console:develop.features.authenticationProvider.forms.common" +
+                ".requiredErrorMessage");
+        }
+        if (!values.appId) {
+            errors.appId = t("console:develop.features.authenticationProvider.forms.common" +
+                ".requiredErrorMessage");
+        }
+        if (!values.apiToken) {
+            errors.apiToken = t("console:develop.features.authenticationProvider.forms.common" +
+                ".requiredErrorMessage");
+        }
+        if (!values.baseUrl) {
+            errors.baseUrl = t("console:develop.features.authenticationProvider.forms.common" +
+                ".requiredErrorMessage");
+        }
+
+        return errors;
+    };
+
+    return (
+        (isIdPListRequestLoading !== undefined && isIdPListRequestLoading === false)
+            ? (
+                <Wizard
+                    id={ FORM_ID }
+                    initialValues={ { name: template?.idp?.name } }
+                    onSubmit={
+                        (values: HyprAuthenticationProviderCreateWizardFormValuesInterface) => onSubmit(values)
+                    }
+                    triggerSubmit={ (submitFunction: any) => triggerSubmission(submitFunction) }
```

Comment:
Can we avoid the `any` data type and use the correct data type

Reply:
addressed with [f58871f](https://github.com/wso2/identity-apps/pull/3790/commits/f58871f9724393ba68ae73f009cf80d4cd28512f)